CONTRIBUTING.md

```diff
@@ -15,50 +15,56 @@ To develop PyG on your machine, here are some tips:
 
 1. Uninstall all existing PyG installations:
 
-    ```
-    pip uninstall torch-geometric
-    pip uninstall torch-geometric  # run this command twice
-    ```
+   ```bash
+   pip uninstall torch-geometric
+   pip uninstall torch-geometric  # run this command twice
+   ```
 
 2. Clone a copy of PyG from source:
 
-    ```
-    git clone https://github.com/pyg-team/pytorch_geometric
-    cd pytorch_geometric
-    ```
+   ```bash
+   git clone https://github.com/pyg-team/pytorch_geometric
+   cd pytorch_geometric
+   ```
 
 3. If you already cloned PyG from source, update it:
 
-    ```
-    git pull
-    ```
+   ```bash
+   git pull
+   ```
 
 4. Install PyG in editable mode:
 
-    ```
-    pip install -e .[test]
-    ```
+   ```bash
+   pip install -e .[dev]
+   ```
 
-    This mode will symlink the Python files from the current local source tree into the Python install. Hence, if you modify a Python file, you do not need to reinstall PyG again and again.
+   This mode will symlink the Python files from the current local source tree into the Python install. Hence, if you modify a Python file, you do not need to reinstall PyG again and again.
 
 5. Ensure that you have a working PyG installation by running the entire test suite with
 
-    ```
-    pytest
-    ```
+   ```bash
+   pytest
+   ```
 
 In case an error occurs, please first check if all sub-packages ([`torch-scatter`](https://github.com/rusty1s/pytorch_scatter), [`torch-sparse`](https://github.com/rusty1s/pytorch_sparse), [`torch-cluster`](https://github.com/rusty1s/pytorch_cluster) and [`torch-spline-conv`](https://github.com/rusty1s/pytorch_spline_conv)) are on its latest reported version.
```

Comment:
nit:
```suggestion
   In case an error occurs, please first check if all sub-packages ([`torch-scatter`](https://github.com/rusty1s/pytorch_scatter), [`torch-sparse`](https://github.com/rusty1s/pytorch_sparse), [`torch-cluster`](https://github.com/rusty1s/pytorch_cluster) and [`torch-spline-conv`](https://github.com/rusty1s/pytorch_spline_conv)) are on its latest reported version.
```